integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestGlobalSortDataLoad.scala

```diff
@@ -480,6 +482,19 @@ class TestGlobalSortDataLoad extends QueryTest with BeforeAndAfterEach with Befo
     checkAnswer(sql("select * from sink"), Row("k", null, null,null,null, null))
   }
 
+  test("test decimal and complex value as null with global sort load") {
```

Comment:
no need to create a new test case, @kunal642 already added a test case called `test global sort with null values`, please include these there only and just add Jira number in the comment of the test case.

Reply:
OK